BenchmarkDotNet/Jobs/Job.cs

```diff
@@ -16,7 +15,12 @@ public class Job : IJob
         public static readonly IJob Mono = new Job { Runtime = Runtime.Mono };
         public static readonly IJob LongRun = new Job { LaunchCount = 3, WarmupCount = 30, TargetCount = 1000 };
 
-        public IToolchain Toolchain { get; set; } = ClassicToolchain.Instance;
+        public IToolchain Toolchain { get; set; }
+#if CLASSIC
+            = Toolchains.Classic.ClassicToolchain.Instance;
+#elif DNX
+            = Toolchains.Dnx.DnxToolchain.Instance;
+#endif
```

Comment:
Can we put current configuration (`CLASSIC`/`DNX`) in a global constant and use it to write nice code without `#if`/`#elif`/`#endif`?